docs/_components.scss

```diff
@@ -803,4 +803,84 @@
     100% {
         @include transform(rotate(360deg));
     }
+}
+
+// Page Footer
+//
+// markup:
+// <footer>
+//      <ul class="footer-links columns is-mobile">
+//          <li class="column"><a href="#">About</a></li>
+//          <li class="column"><a href="#">FAQ</a></li>
+//          <li class="column"><a href="#">Contact</a></li>
+//          <li class="column"><a href="#">Privacy</a></li>
+//      </ul>
+//      <h4 class='footer-social__heading'>Follow us</h4>
+//      <ul class="footer-social columns is-mobile">
+//          <div class="column">
+//              <li><a href="#"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 400 400" class="icon_twitter"><path class="icon_twitter--icon" d="M199.85.44c-110.46 0-200 89.54-200 200s89.54 200 200 200 200-89.54 200-200-89.54-200-200-200zm45.2 104.5c13.87 0 26.4 5.87 35.2 15.24 10.98-2.16 21.3-6.18 30.6-11.7-3.6 11.25-11.23 20.7-21.2 26.67 9.76-1.1 19.06-3.7 27.7-7.5-6.46 9.68-14.64 18.17-24.05 24.98.1 2.07.14 4.15.14 6.23 0 63.7-48.5 137.2-137.2 137.2-27.3 0-52.6-8-73.9-21.66 3.7.44 7.6.66 11.5.66 22.6 0 43.3-7.7 59.8-20.65-21.1-.4-38.9-14.3-45.06-33.5 2.94.6 5.96.9 9.06.9 4.4 0 8.67-.6 12.7-1.7-22.04-4.4-38.7-23.9-38.7-47.22 0-.22 0-.42.05-.62 6.5 3.64 13.9 5.8 21.8 6.05-12.94-8.6-21.45-23.4-21.45-40.1 0-8.83 2.36-17.1 6.5-24.23 23.77 29.18 59.3 48.37 99.37 50.38-.85-3.5-1.28-7.2-1.28-10.97 0-26.64 21.6-48.2 48.2-48.2z" /></svg></a></li>
+//              <li><a href="#"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 60 60" class="icon_facebook"><path class="icon_facebook--icon" d="M25.77 46.32h6.74V30H37l.6-5.63h-5.1l.02-2.8c0-1.48.14-2.26 2.24-2.26h2.8v-5.6h-4.5c-5.4 0-7.3 2.73-7.3 7.32v3.37H22.4V30h3.36v16.32zM30 60C13.43 60 0 46.57 0 30S13.43 0 30 0s30 13.43 30 30-13.43 30-30 30z" clip-rule="evenodd" fill-rule="evenodd"/></svg></a></li>
+//              <li><a href="#"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1000 1000" class="icon_linkedin"><path class="icon_linkedin--icon" d="M500 10c136.1 0 251.8 47.6 347.1 142.9C942.4 248.2 990 363.9 990 500c0 134.7-47.6 250.1-142.9 346.1C751.8 942 636.1 990 500 990c-134.8 0-250.1-48-346.1-143.9C58 750.1 10 634.8 10 500c0-136.1 48-251.8 143.9-347.1C249.9 57.6 365.3 10 500 10M379.5 702.1V383.6h-98v318.5h98m-49-359.3c35.4 0 53.1-16.3 53.1-49s-17.7-49-53.1-49c-15 0-27.6 4.8-37.8 14.3-10.2 9.5-15.3 21.1-15.3 34.7.1 32.7 17.8 49 53.1 49M743 702.1V514.3c0-44.9-10.5-79.3-31.6-103.1-21.1-23.8-48.7-35.7-82.7-35.7-44.9 0-77.6 17-98 51h-2l-6.1-42.9h-85.8c1.4 19.1 2 53.1 2 102.1v216.4h100V518.4c0-12.3 1.4-20.4 4.1-24.5 8.2-25.9 25.2-38.8 51-38.8 32.7 0 49 23.1 49 69.4v177.6H743"/></svg></a></li>
+//          </div>
+//      </ul>
+// </footer>
+// 
+// Styleguide 5.8
+footer {
+    background: #1C293B;
+    padding: 20px;
```

Comment:
As a general rule I've been setting padding and margins based on the $baseline variable of 1.5rem. You'll see lots of things like `$baseline / 2` or `$baseline * 2` in the other components. It just keeps the rhythm of the page consistent and means we can scale things proportionally by changing one variable. Its particularly important for top/bottom margins as all the typography is based on 1.5rem lines. Its not applicable in all cases (not sure here for example) but its worth bearing in mind that you should use $baseline whenever possible.